eng/Signing.props

```diff
@@ -15,7 +15,11 @@
   <!--
     Non-default certificates.
   -->
-  <ItemGroup>
+  <ItemGroup>    
+    <!-- Exclude two manually zipped servicehub symbol nupkgs from being signed -->
+    <ItemsToSign Remove="@(ItemsToSign)" Condition="'%(FileName)%(Extension)' == 'Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.arm64.symbols.nupkg'" />
```

Comment:
You should be able to use an absolute path:

`<ItemsToSign Remove="$(ArtifactsNonShippingPackagesDir)Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.arm64.symbols.nupkg"